Core/src/org/sleuthkit/autopsy/commonfilesearch/CommonFilesPanel.java

```diff
@@ -237,8 +247,8 @@ protected CommonFilesMetadata doInBackground() throws TskCoreException, NoCurren
             protected void done() {
                 try {
                     super.done();
-
-                    CommonFilesMetadata metadata = get();
+                    progress.setDisplayName(Bundle.CommonFilesPanel_search_done_searchProgress2());
```

Comment:
I don't think that the difference between "gathering results" and "generating results" says much. I also suspect that the code between the generating and the displaying messages runs very quickly. Have you experienced otherwise?

Reply:
Removed the second text change.